public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx

```diff
@@ -212,38 +211,22 @@ export const VisualizationContainer = ({
   }
 
   const loadVisaulization = async () => {
-    if (catalogVisualization)
-      await renderCatalogVisualization({
-        http,
-        pplService,
-        catalogSource: savedVisualizationId,
-        startTime: fromTime,
-        endTime: toTime,
-        filterQuery: pplFilterValue,
-        spanParam,
-        setVisualizationTitle,
-        setVisualizationType,
-        setVisualizationData,
-        setVisualizationMetaData,
-        setIsLoading,
-        setIsError,
-      });
-    else
-      await renderSavedVisualization(
-        http,
-        pplService,
-        savedVisualizationId,
-        fromTime,
-        toTime,
-        pplFilterValue,
-        spanParam,
-        setVisualizationTitle,
-        setVisualizationType,
-        setVisualizationData,
-        setVisualizationMetaData,
-        setIsLoading,
-        setIsError
-      );
+    await renderSavedVisualization({
```

Comment:
Argument list converted to object to avoid variable-placement errors in long argument list.